guestbook/README.md

```diff
@@ -4,704 +4,303 @@
-[gce-firewall-docs]: https://cloud.google.com/compute/docs/networking#firewalls
+### Viewing the Frontend Service via `LoadBalancer`
+
+Once the frontend Service is running, you need to find the IP address to view your Guestbook.
+
+1. Run the following command to get the IP address for the frontend Service.
+
+       kubectl get service frontend
+
+   The response should be similar to this:
+
+       NAME       CLUSTER-IP      EXTERNAL-IP        PORT(S)        AGE
+       frontend   10.51.242.136   109.197.92.229     80:32372/TCP   1m
+
+2. Copy the External IP address, and load the page.
+
+## Scale the Web Frontend 
+
+Scaling up or down is easy because your servers are defined as a Service that uses a Deployment controller.
+
+1. Run the following command to scale up the number of frontend Pods:
+
+       kubectl scale deployment frontend --replicas=5
+
+2. Query the list of Pods to verify the number of frontend Pods running:
+
+       kubectl get pods
+
+   The response should look similar to this: 
+
+       NAME                            READY     STATUS    RESTARTS   AGE
+       frontend-3823415956-70qj5       1/1       Running   0          5s
+       frontend-3823415956-dsvc5       1/1       Running   0          54m
+       frontend-3823415956-k22zn       1/1       Running   0          54m
+       frontend-3823415956-w9gbt       1/1       Running   0          54m
+       frontend-3823415956-x2pld       1/1       Running   0          5s
+       redis-master-1068406935-3lswp   1/1       Running   0          56m
+       redis-slave-2005841000-fpvqc    1/1       Running   0          55m
+       redis-slave-2005841000-phfv9    1/1       Running   0          55m
+
+3. Run the following command to scale down the number of frontend Pods:
+
+       kubectl scale deployment frontend --replicas=2
+
+4. Query the list of Pods to verify the number of frontend Pods running:
+
+       kubectl get pods
+
+   The response should look similar to this:
+
+       NAME                            READY     STATUS    RESTARTS   AGE
+       frontend-3823415956-k22zn       1/1       Running   0          1h
+       frontend-3823415956-w9gbt       1/1       Running   0          1h
+       redis-master-1068406935-3lswp   1/1       Running   0          1h
+       redis-slave-2005841000-fpvqc    1/1       Running   0          1h
+       redis-slave-2005841000-phfv9    1/1       Running   0          1h
+        
+{% endcapture %}
+
+{% capture cleanup %}
+Deleting the Deployments and Services also deletes any running Pods. Use labels to delete multiple resources with one command.
+
+1. Run the following commands to delete all Pods, Deployments, and Services.
+
+       kubectl delete deployment -l app=redis
+       kubectl delete service -l app=redis
+       kubectl delete deployment -l app=guestbook
+       kubectl delete service -l app=guestbook
+
+   The responses should be:
+
+       deployment "redis-master" deleted
+       deployment "redis-slave" deleted
+
```

Comment:
you can delete these empty lines in the output, it looks more confusing IMO. Also there won't be empty lines in the output if I copypaste the commands above